tests/test_job.py

```diff
@@ -432,6 +432,35 @@ def test_description_is_persisted(self):
         Job.fetch(job.id, connection=self.testconn)
         self.assertEqual(job.description, "tests.fixtures.say_hello('Lionel')")
 
+    def test_multiple_dependencies_are_accepted_and_persisted(self):
+        """Ensure job._dependency_ids accepts different input formats, and
+        is set and restored properly"""
+        jobA = Job.create(func=fixtures.some_calculation, args=(3, 1, 4), id="A")
```

Comment:
Let's use snake case instead of camel case to keep things consistent.

Reply:
Done.